config/pipeline.yaml

```diff
@@ -101,6 +101,22 @@ storage:
 
 runtimes:
 
+  lab-kontron:
+    lab_type: legacy.lava_rest
+    url: 'https://lavalab.kontron.com/'
+    filters:
+      - passlist:
+          plan:
+            - baseline
+
+  lab-pengutronix:
+    lab_type: legacy.lava_rest
```

Comment:
No sure the new system implements this lab_type, @nuclearcat ?

Reply:
It doesn't - that's a `lab_type` from the legacy system. New one relies on `runtime` names which would be just `lava` in this case.